apiserver/facades/client/modelmanager/services.go

```diff
@@ -84,7 +84,7 @@ type ModelDefaultsService interface {
 type ModelInfoService interface {
 	// CreateModel is responsible for creating a new read only model
 	// that is being imported.
-	CreateModel(context.Context, coremodel.UUID, uuid.UUID) error
+	CreateModel(context.Context, uuid.UUID) error
```

Comment:
This breaks a lot of things :facepalm: We need the controller uuid otherwise k8s doesn't annotate correctly.

Reply:
This is still the controller uuid. I just took it off the model id type as that name will change and it will just be confusing.

Reply:
:disappointed: